### PR TITLE
Pull RSS feed from blog for events

### PR DESCRIPTION
**Pull request by endorama**
#### Summary

Add logic to events page that pulls RSS information from blog.sigstore.dev

There are some implementation choices open for discussion:
- using `axios`, for fetching content
- leveraging `api.rss2json.com`, to transform XML feed to JSON
- changing `date` to `pubDate` in events markdown frontmatters, to align with data from the RSS feed and allow sorting more easily

Not sure is this PR needs a release note (PR is editable by maintainers in case).

#### Ticket Link

Closes https://github.com/sigstore/sigstore-website/issues/114

#### Release Note

```release-note

```

### CODE REVIEW

**Comment by bobcallaway** [comment]
Looks great, thanks @endorama !

Two minor questions: 
1. Should we limit the list to the last 5 posts?
2. I noticed in one of the titles we had `&amp;` instead of `&`... is there some escaping we need to do on the titles?

**Comment by mdunbavan** [comment]
@bobcallaway @endorama I think the limit to 5 posts is a great idea. When we initially built this we had that setup as a possibility but there wasn't scope for lots of articles being written but now sigstore is gaining traction articles are coming in thick and fast :)

Great work btw @endorama

File: components/newsEvents.vue [hunk 1]
```diff
@@ -70,10 +70,30 @@ export default {
...
-            const articles = globalData;
+
+            let articlesFromBlog = [];
+            try {
+                const results = await this.$axios.$get("https://api.rss2json.com/v1/api.json?rss_url=https://blog.sigstore.dev/feed");
+                articlesFromBlog = results.items;
+            } catch {
+                console.error("Cannot retrieve blog RSS feed");
+            }
+
```
Comment (mdunbavan) [inline components/newsEvents.vue:81]: @endorama I think pagination should reside in here somewhere

**Review by mdunbavan — CHANGES_REQUESTED** [review]
If we add the pagination I think we are good to go

**Comment by endorama** [comment]
Thanks all!

> Should we limit the list to the last 5 posts?

I think for sure in the home page, but what about in the community page? Should we add pagination to both or limit to 5 for home page and paginate (maybe at 10) in community page?

> I noticed in one of the titles we had &amp; instead of &... is there some escaping we need to do on the titles?

Probably the XML to JSON conversion doing weird things, I'll have a look.

**Comment by endorama** [comment]
I added pagination, the page number style is very basic, comments welcome! I have to refine it a bit (as there is no logic limiting number of displayed page this will go wild when there will a lot of articles) but the general implementation and design is there.
I added it only to the community page. 

What do you think?

**Comment by endorama** [comment]
I was also wondering if there was a way to fetch the feed list at build time instead that requesting them for each visitor (not our APIs but free version is limited + lot of useless traffic). I think so but my experience with NuxtJS was zero before this PR. so I'm not really a pro :)

**Comment by bobcallaway** [comment]
LGTM - I love how it folds in the RSS feed with the ability to still specify manual news stories.

**Review by lukehinds — APPROVED** [review]

**Comment by lukehinds** [comment]
@mdunbavan this still has you as 'requested changes', is this good now?

**Review by mdunbavan — APPROVED** [review]

**Pull request merged**